db/migrate/20230821035222_create_order_items.rb

```diff
@@ -0,0 +1,12 @@
+class CreateOrderItems < ActiveRecord::Migration[6.1]
+  def change
+    create_table :order_items do |t|
+      t.integer :order_id
```

Comment:
```suggestion
      t.references :order
```
quan hệ dùng cái này đi em ERD còn có liên kết
tham khảo: https://medium.com/@where_is_x/activerecord-t-references-foreign-key-and-belongs-to-e58cebc79493